### ISSUE TITLE
[WASM] apps/wasm-standalone is broken

### ISSUE THREAD

**Issue by WenheLI** [apache/tvm#6817]
I was trying to run `apps/wasm-standalone` and this does not work for me right now.
While running ` python3 ./build_graph_lib.py -O3 ./resnet50v1.onnx  `, I got the following errors:
```bash
WARNING:autotvm:Cannot find config for target=llvm -keys=cpu -mattr=+simd128 -mtriple=wasm32-unknown-unknown -system-lib=1, workload=('dense_nopack.x86', ('TENSOR', (1, 2048), 'float32'), ('TENSOR', (1000, 2048), 'float32'), None, 'float32'). A fallback configuration is used, which may bring great performance regression.
Traceback (most recent call last):
  File "./build_graph_lib.py", line 78, in <module>
    build_graph_lib(args.model_file, args.opt_level)
  File "./build_graph_lib.py", line 49, in build_graph_lib
    graph_json, lib, params = relay.build(mod, target=target, params=params)
  File "/home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/relay/build_module.py", line 255, in build
    graph_json, mod, params = bld_mod.build(mod, target, target_host, params)
  File "/home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/relay/build_module.py", line 121, in build
    self._build(mod, target, target_host)
  File "tvm/_ffi/_cython/./packed_func.pxi", line 321, in tvm._ffi._cy3.core.PackedFuncBase.__call__
  File "tvm/_ffi/_cython/./packed_func.pxi", line 256, in tvm._ffi._cy3.core.FuncCall
  File "tvm/_ffi/_cython/./packed_func.pxi", line 245, in tvm._ffi._cy3.core.FuncCall3
  File "tvm/_ffi/_cython/./base.pxi", line 160, in tvm._ffi._cy3.core.CALL
tvm._ffi.base.TVMError: Traceback (most recent call last):
  [bt] (8) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(std::_Function_handler<void (tvm::runtime::TVMArgs, tvm::runtime::TVMRetValue*), tvm::relay::backend::RelayBuildModule::GetFunction(std::string const&, tvm::runtime::ObjectPtr<tvm::runtime::Object> const&)::{lambda(tvm::runtime::TVMArgs, tvm::runtime::TVMRetValue*)#3}>::_M_invoke(std::_Any_data const&, tvm::runtime::TVMArgs&&, tvm::runtime::TVMRetValue*&&)+0x10e) [0x7f4ae6da26ce]
  [bt] (7) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::relay::backend::RelayBuildModule::BuildRelay(tvm::IRModule, std::unordered_map<std::string, tvm::runtime::NDArray, std::hash<std::string>, std::equal_to<std::string>, std::allocator<std::pair<std::string const, tvm::runtime::NDArray> > > const&)+0x9c1) [0x7f4ae6da1ae1]
  [bt] (6) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::build(tvm::Map<tvm::runtime::String, tvm::IRModule, void, void> const&, tvm::Target const&)+0x584) [0x7f4ae65c9af4]
  [bt] (5) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::build(tvm::Map<tvm::Target, tvm::IRModule, void, void> const&, tvm::Target const&)+0x403) [0x7f4ae65c9093]
  [bt] (4) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::codegen::Build(tvm::IRModule, tvm::Target)+0x281) [0x7f4ae69c3081]
  [bt] (3) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(+0x151df00) [0x7f4ae6f02f00]
  [bt] (2) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::codegen::LLVMModuleNode::Init(tvm::IRModule const&, tvm::Target const&)+0x41) [0x7f4ae6f056c1]
  [bt] (1) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(tvm::codegen::GetLLVMTargetMachine(tvm::Target const&, bool)+0x4a2) [0x7f4ae6effd92]
  [bt] (0) /home/codespace/workspace/incubator-tvm/pythonenv3.8/lib/python3.8/site-packages/tvm/libtvm.so(dmlc::LogMessageFatal::~LogMessageFatal()+0x6d) [0x7f4ae6513a7d]
  File "/workspace/tvm/src/target/llvm/llvm_common.cc", line 136
TVMError: Check failed: allow_null: No available targets are compatible with triple "wasm32-unknown-unknown" target_triple=wasm32-unknown-unknown
```

Any thoughts on how to fix it?

**Comment by leonwanghui** [comment]
@WenheLI Sorry for the late reply, this issue has been addressed by PR #6862, please download the latest version of code and try it again.

**Comment by junrushao1994** [comment]
Closed via #6862. Please feel free to reopen if the bug still exists. Thanks!
